examples/Cassie/osc_walk/heading_traj_generator.cc

```diff
@@ -0,0 +1,101 @@
+#include "examples/Cassie/osc_walk/heading_traj_generator.h"
+
+#include <math.h>
+#include <string>
+
+#include "attic/multibody/rigidbody_utils.h"
+
+using std::cout;
+using std::endl;
+using std::string;
+
+using Eigen::MatrixXd;
+using Eigen::Vector2d;
+using Eigen::Vector3d;
+using Eigen::VectorXd;
+
+using dairlib::systems::OutputVector;
+
+using drake::systems::BasicVector;
+using drake::systems::Context;
+using drake::systems::LeafSystem;
+
+using drake::trajectories::PiecewisePolynomial;
+
+namespace dairlib {
+namespace cassie {
+namespace osc_walk {
+
+HeadingTrajGenerator::HeadingTrajGenerator(const RigidBodyTree<double>& tree,
+                                           int pelvis_idx)
+    : tree_(tree), pelvis_idx_(pelvis_idx) {
+  // Input/Output Setup
+  state_port_ = this
+                    ->DeclareVectorInputPort(OutputVector<double>(
```

Comment:
Minor comment: Drake's autoformat seems to have this spacing issue. I'll take a look to see if I can modify the .clang_format file to fix this.